source/request-as-event-emitter.ts

```diff
@@ -246,8 +254,8 @@ export default (options, input?: TransformStream) => {
 		} else {
 			// Catches errors thrown by calling requestFn(...)
 			try {
-				// TODO: Properly type this.
-				handleRequest(requestFn(options as https.RequestOptions, handleResponse));
+				// @ts-ignore Something's up with ts
```

Comment:
??? Can you elaborate?

Reply:
```
Argument of type 'URL' is not assignable to parameter of type 'string | URL'.
  Type 'URL' is not assignable to type 'import("url").URL'.
    Types of property 'searchParams' are incompatible.
      Type 'URLSearchParams' is missing the following properties from type 'URLSearchParams': entries, keys, values, [Symbol.iterator]ts(2345)
```
Casting it as an url, but it seems to... dislike it? Which is why the @ts-ignore

Reply:
Seems like a TS bug.